### DISCUSSION TITLE
24h format option

### DISCUSSION THREAD

**Discussion by arlexTech** [alexjustesen/speedtest-tracker#1765] [Ideas]
hello,
is it possible to add an option to switch to 24 hours format ?

**Comment by svenvg93** [comment]
You can use the Environment Variables `CHART_DATETIME_FORMAT` or `DATETIME_FORMAT ` to set the format to your liking. 
Check out https://www.php.net/manual/en/datetime.format.php to see how to specify the format. 

For example I use the format: `18 Oct 2024, 16:06:00`

By setting this in my environment 

```
DATETIME_FORMAT: "j M Y, G:i:s"
CHART_DATETIME_FORMAT: "j/m G:i"
```

Link to the docs; https://docs.speedtest-tracker.dev/getting-started/environment-variables
